config/samples/enterprisesearch/ent_es.yml

```diff
@@ -16,9 +16,9 @@ spec:
 apiVersion: enterprisesearch.k8s.elastic.co/v1beta1
 kind: EnterpriseSearch
 metadata:
-  name: entsearch-sample
+  name: ent-sample
 spec:
-  version: 7.6.0
+  version: 7.7.0
```

Comment:
I would keep this upgrade out of this PR or bump the version everywhere.

Reply:
you're right let's do this in a different PR.